rotkehlchen/db/upgrades/v31_v32.py

```diff
@@ -86,6 +87,52 @@ def _add_new_tables(cursor: 'Cursor') -> None:
 );""")  # noqa: E501
 
 
+def _update_history_entries_from_kraken(cursor: 'Cursor') -> None:
+    """The logic for kraken was adding additional entries for trades when fee + kfee was
+    being used. This function makes the state of the database consistent with the upgraded
+    logic by:
+    - Removing extra row additions
+    - Updating the sequence index to avoid duplicates
+    - Make sure that no other event has duplicated sequence indexes
+    """
+    cursor.execute("""
+    DELETE FROM history_events where location="B" AND asset="KFEE" AND
+     type="trade" AND subtype=NULL;
+    """)
+    cursor.execute("""
+    UPDATE history_events SET sequence_index = sequence_index + 1 WHERE asset="KFEE"
```

Comment:
What if this creates a conflict? Why is this line here?

Reply:
I just removed it. Wanted to make sure that those are removed but indeed the other logic is enough